integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/database/Database.java

```diff
@@ -0,0 +1,86 @@
+package com.sequenceiq.it.cloudbreak.newway.entity.database;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+import com.sequenceiq.it.IntegrationTestContext;
+import com.sequenceiq.it.cloudbreak.newway.Assertion;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.GherkinTest;
+import com.sequenceiq.it.cloudbreak.newway.Prototype;
+import com.sequenceiq.it.cloudbreak.newway.ResourceAction;
+import com.sequenceiq.it.cloudbreak.newway.action.Action;
+import com.sequenceiq.it.cloudbreak.newway.action.database.DatabaseDeleteAction;
+import com.sequenceiq.it.cloudbreak.newway.action.database.DatabaseListAction;
+import com.sequenceiq.it.cloudbreak.newway.action.database.DatabaseCreateAction;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+import com.sequenceiq.it.cloudbreak.newway.v4.DatabaseAction;
+
+@Prototype
+public class Database extends DatabaseEntity {
```

Comment:
Is it a renaming or new class?

Reply:
its a renaming

Reply:
Is it possible to remove? We should remove all old *Entity children classes like Database